src/main/java/seedu/address/logic/commands/eventcommands/AddPollCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.commands.eventcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Command to add a new poll to the pre-selected event.
+ */
+public class AddPollCommand extends Command {
+
+    public static final String COMMAND_WORD = "addPoll";
+
```

Comment:
Redundant space

Reply:
Actually the given code uses a lot of newlines as well. Is there a rule against this?

Reply:
No rule against it. It just looks weird to have a new line for no reason. I guess it's alright even if we just leave it as it is.